packages/performance-tests/test/TransformerRegression.test.ts

```diff
@@ -156,13 +175,16 @@ async function runRegressionTests() {
           sourceDb.close(); // closing to ensure connection cache reusage doesn't affect results
         });
 
-        testCasesMap.forEach(async (testCase, key) => {
-          it(key, async () => {
-            await testCase(sourceDb,
-              (...smallReportSubset: [testName: string, iModelName: string, valDescription: string, value: number]) => {
-                reporter.addEntry( ...smallReportSubset, reportInfo );
-              });
-          }).timeout(0);
+        transformerModules.forEach((transformerModule: TestTransformerModule) => {
```

Comment:
let's flip the order of these two for loops, ideally we run every transformer Module for each test case, then move onto the next test case